### ISSUE TITLE
aarch64 for armv81

### ISSUE THREAD

**Issue by varyaggit** [termux/proot-distro#34]
hi.
I saw that in scripts, fore example ./archlinux.sh,  in case armv81 will install  ArchLinuxARN-armv7.
Why not aarch64?
armv81 is aarch64  or i am wrong?
If so can we update the scripts and use aarch64 for armv81?

**Comment by xeffyr** [comment]
> armv81 is aarch64 or i am wrong?

armv8l - ARM 32bit operating system on 64bit CPU. Since Termux is 32bit here, proot is 32bit as well. So you have no choice.

**Comment by varyaggit** [comment]
You mean that my termux version is on 32 bit?
do i understand well?

**Comment by xeffyr** [comment]
Yes, of course, your Termux version is 32 bit. Otherwise `uname -m` would not report `armv8l`.

Some manufacturers do not bother to provide 64bit Android (which normally consists of both 64 and 32 bit library set) for their devices, one of reasons is a small size of `/system` partition.

**Comment by xeffyr** [comment]
Since Termux heavily relies on Android core libraries (libc, libm, liblog, etc), we have to assume that `armv8l` equals to `armv7l`.

**Comment by varyaggit** [comment]
Thank you for your time and explanation. Actually i have Huawei  Mediapad M6. 
I installed termux v 0.98 from apkpure.
 uname -m was armv81.
I uninstalled this version of termux. Downloaded and installed v 0.102 from f-droid and my 
uname -m is aarch64.
On another version v.0.94 downloaded from Huawei app galery output of uname -m was aarch64.
Only apk from apkpure strangly was armv81. 
Спасибо за помощь.
